go.mod

```diff
@@ -0,0 +1,61 @@
+module github.com/docker/libkv
+
+go 1.17
+
+require (
+	github.com/coreos/etcd v3.3.25+incompatible
+	github.com/hashicorp/consul/api v1.1.0
+	github.com/samuel/go-zookeeper v0.0.0-20201211165307-7117e9ea2414
```

Comment:
Comparing this one as well https://github.com/samuel/go-zookeeper/compare/d0e0d8e11f318e000a8cc434616d69e329edc374...7117e9ea2414